airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -26,6 +26,55 @@
 Changelog
 ---------
 
+8.20.0
+......
+
+Features
+~~~~~~~~
+
+* ``AWS Batch Executor (#37618)``
+* ``Add fallback 'region_name' value to AWS Executors (#38704)``
+* ``Introduce Amazon Bedrock service (#38602)``
+* ``Amazon Bedrock - Model Customization Jobs (#38693)``
+* ``ECS Executor - add support to adopt orphaned tasks. (#37786)``
+* ``Update AWS auth manager CLI command to not disable AVP schema validation (#38301)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Reduce 's3hook' memory usage (#37886)``
+* ``Add check in AWS auth manager to check if the Amazon Verified Permissions schema is up to date (#38333)``
+* ``fix: COMMAND string should be raw to avoid SyntaxWarning: invalid escape sequence '\s' (#38734)``
```

Comment:
It's been reverted

```suggestion
```

Reply:
This entry will be moved to hidden from log when I refresh the docs before merge

Reply:
https://github.com/apache/airflow/pull/38863/commits/df5ae10b80490d4cd382c9a028a393b10f136c4b